pkg/monitoring/metrics/virt-controller/vmsnapshot_test.go

```diff
@@ -0,0 +1,52 @@
+package virt_controller_test
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	. "github.com/onsi/ginkgo/v2"
	. "github.com/onsi/gomega"

        corev1 "k8s.io/api/core/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

similar imports should be grouped